pytest_factoryboy/plugin.py

```diff
@@ -58,4 +58,4 @@ def pytest_runtest_call(item):
 def pytest_addhooks(pluginmanager):
     """Register plugin hooks."""
     from pytest_factoryboy import hooks
-    pluginmanager.addhooks(hooks)
+    pluginmanager.add_hookspecs(hooks)
```

Comment:
You're right, I can update that too if you'd like. But there's no rush anymore, 2.8.1 should work.

I don't actually use factoryboy personally, I only put this up because it helped me triage that pytest bug.